frontend/src/Components/Output.jsx

```diff
@@ -0,0 +1,113 @@
+import { useState } from "react";
+import { Light as SyntaxHighlighter } from "react-syntax-highlighter";
+import { docco,github,monokai } from "react-syntax-highlighter/dist/esm/styles/hljs";
+
+const Output = () => {
+  const [option, setOption] = useState("Output");
+  const [compilation, setCompilation] = useState("Not Started");
+  const [time, setTime] = useState(0);
+  const [error, setError] = useState(false);
+  const [errorCount, setErrorCount] = useState(0);
+  const [warning, setWarning] = useState(0);
+    const [errors, setErrors] = useState("");
+
+    const [output, setOutput] = useState('Not Compiled');
+    const [files, setFiles] = useState('Still Not Compiled');
+    
+    let errorText = `Warning: Each child in a list should have a unique "key" prop.
+
+Check the render method of App. See https://reactjs.org/link/warning-keys for more information.
+`;
+    
+    let outputText = `Ramcharan`
+
+  const handleChange = (event) => {
+      setOption(event.target.value);
+    //   if (event.target.value == 'Graphs') {
```

Comment:
Is this commented code required?

Reply:
No Sir that just to show outputs
I will remove in next commit.